neutron/tests/unit/agent/test_securitygroups_rpc.py

```diff
@@ -1111,6 +1111,7 @@ class BaseSecurityGroupAgentRpcTestCase(base.BaseTestCase):
     def setUp(self, defer_refresh_firewall=False):
         super(BaseSecurityGroupAgentRpcTestCase, self).setUp()
         set_firewall_driver(FIREWALL_NOOP_DRIVER)
+        ipv6._IS_IPV6_ENABLED = True
```

Comment:
I agree, especially for the submission to the community. Most of the tests did pass before I put that line in the code. The ones that didn't were because the test had a hack that forced iptables_manager to ipv6, which didn't work once I differentiated iptable management between ipv4/6.